gymnasium/wrappers/__init__.py

```diff
@@ -15,7 +15,7 @@
     Box(-1.0, 1.0, (3,), float32)
     >>> wrapped_env = RescaleAction(base_env, min_action=0, max_action=1)
     >>> wrapped_env.action_space
-    Box(-1.0, 1.0, (3,), float32)
+    Box(0.0, 1.0, (3,), float32)
```

Comment:
why was this changed?

Reply:
https://github.com/Farama-Foundation/Gymnasium/actions/runs/5356895864

It was my understanding that to resolve the error that caused this to fail, I would need to change the docstring in init to accurately reflect the changes made in this PR

Reply:
The previous version of line 18 demonstrates the previous functionality where the action space of the env is not updated with the min and max values

Reply:
nevermind, my bad i misread it

Reply:
No worries